fossunited/chapters/doctype/foss_chapter/test_foss_chapter.py

```diff
@@ -1,8 +1,95 @@
 # Copyright (c) 2023, Frappe x FOSSUnited and Contributors
 # See license.txt
 
+import frappe
+from faker import Faker
 from frappe.tests.utils import FrappeTestCase
 
 
 class TestFOSSChapter(FrappeTestCase):
-    pass
+    def setUp(self):
+        fake = Faker()
+
+        chapter = frappe.get_doc(
+            {
+                "doctype": "FOSS Chapter",
+                "chapter_name": fake.text(max_nb_chars=40),
+                "chapter_type": "FOSS Club",
+                "city": "Pune",
+                "country": "India",
+                "email": fake.email(),
+                "facebook": fake.url(),
+                "instagram": fake.url(),
+                "linkedin": fake.url(),
+                "mastodon": fake.url(),
+                "public_chat_group_url": fake.url(),
+                "state": "Maharashtra",
+                "x": fake.url(),
+            }
+        )
+        chapter.insert()
+
+        if not frappe.db.exists("Role", "Chapter Team Member"):
+            frappe.get_doc({"doctype": "Role", "role_name": "Chapter Team Member"}).insert(ignore_permissions=True)
+        if not frappe.db.exists("Role", "Chapter Lead"):
+            frappe.get_doc({"doctype": "Role", "role_name": "Chapter Lead"}).insert(ignore_permissions=True)
+
+        lead_profile = frappe.get_doc("FOSS User Profile", {"user": "test@example.com"})
```

Comment:
This test@example.com is assumed to always exist right ?

Reply:
When I ran the tests - it failed for me first, as the users didnt exist. It might be good if we create dummy users here too? For the tests and delete them in tearDown ?

Reply:
Test users are created by frappe if tests are enabled on your bench. They are always created, unless you have explicitly deleted them